### PR TITLE
Added tooltip and indicate selected person

### PR DESCRIPTION
**Pull request by elayabharath**
Please ignore the number of squares, I keep hitting the quota limit with firebase - so reduced the number of documents read

image

### CODE REVIEW

File: lib/interactive.dart [hunk 1]
```diff
@@ -126,6 +133,35 @@ class Interactive {
     return color;
   }
 
+  String _lookupFilterLabels(String filter, String option) {
+    String label = "Unknown";
+    for (var f in _filters) {
+      if (f.value == filter) {
```
Comment (lukechurch) [inline lib/interactive.dart]: Consider inverting this to 

if (f.value != filter) continue;

It'll decrease the overall nesting level that makes the logic hard to read

File: lib/interactive.dart [hunk 2]
```diff
@@ -126,6 +133,35 @@ class Interactive {
...
+    return label;
+  }
+
+  String _lookupThemeLabels(List<String> themes) {
+    List<String> labels = List();
+    for (var theme in themes) {
+      for (var t in _themes) {
+        if (t.value == theme) {
+          labels.add(t.label);
+          continue;
```
Comment (lukechurch) [inline lib/interactive.dart]: the `continue` is superfluous here

File: lib/interactive.dart [hunk 3]
```diff
@@ -126,6 +133,35 @@ class Interactive {
...
   }
 
+  String _lookupFilterLabels(String filter, String option) {
+    String label = "Unknown";
+    for (var f in _filters) {
+      if (f.value == filter) {
+        for (var o in f.options) {
+          if (o.value == option) {
+            label = o.label;
+            continue;
```
Comment (lukechurch) [inline lib/interactive.dart]: Can we return here instead of continuing?

**Review by lukechurch — APPROVED** [review]
Thanks @elayabharath

File: lib/interactive.dart (hunk 2 above)
Comment (elayabharath) [reply lib/interactive.dart]: Sorry I got confused it with break

File: lib/interactive.dart (hunk 1 above)
Comment (elayabharath) [reply lib/interactive.dart]: fixed

File: lib/interactive.dart (hunk 3 above)
Comment (elayabharath) [reply lib/interactive.dart]: fixed

**Pull request merged**